packages/terra-application-utility/src/utility/_UtilityMenuItem.jsx

```diff
@@ -0,0 +1,193 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import Arrange from 'terra-arrange';
+import Button from 'terra-button';
+import IconCheckmark from 'terra-icon/lib/icon/IconCheckmark';
+import IconChevronRight from 'terra-icon/lib/icon/IconChevronRight';
+import Utils from '../_Utils';
+import styles from './_UtilityMenuItem.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The unique key associated with this menu item.
+   */
+  itemKey: PropTypes.string.isRequired,
+  /**
+   * The text to be displayed next to the menu item.
+   */
+  title: PropTypes.string,
+  /**
+   * The component associated with this menu item.
+   */
+  content: PropTypes.element,
+  /**
+   * The location of this menu item.
+   */
+  contentLocation: PropTypes.oneOf([Utils.LOCATIONS.BODY, Utils.LOCATIONS.FOOTER]),
+  /**
+   * Whether this item should be inset to the left. Based on if any other item has isSelected set to true.
+   */
+  leftInset: PropTypes.bool,
+  /**
+   * Whether this item is selected.
+   */
+  isSelected: PropTypes.bool,
+  /**
+   * Whether this item can be toggled.
+   */
+  isSelectable: PropTypes.bool,
+  /**
+   * Whether or not the menu item should display a disclosure indicator.
+   */
+  hasChevron: PropTypes.bool,
+  /**
+   * Function to trigger when this item is selected.
+   */
+  onChange: PropTypes.func.isRequired,
+  /**
+   * Whether this item should be inset to the right. Based on if any other item has a chevron.
+   */
+  rightInset: PropTypes.bool,
+  /**
+   * Sets the Utility variant.
+   */
+  variant: PropTypes.oneOf([Utils.VARIANTS.HEADER, Utils.VARIANTS.MENU]).isRequired,
+};
+
+const defaultProps = {
+  title: '',
+};
+
+class UtilityMenuItem extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleKeyDown = this.handleKeyDown.bind(this);
+    this.state = { isSelected: props.isSelected && props.isSelectable };
```

Comment:
Should this be opened up such that a menu item can be selected but never unselected?